tests/sentry/hybrid_cloud/test_region.py

```diff
@@ -56,10 +56,11 @@ def test_by_organization_slug(self):
     def test_by_organization_id_attribute(self):
         with override_regions(self.regions):
             region_resolution = ByOrganizationIdAttribute("organization_member")
-            org_member = OrganizationMember.objects.create(
-                organization_id=self.organization.id,
-                user_id=self.user.id,
-            )
+            with exempt_from_silo_limits():
```

Comment:
Nothing wrong with this change, _but_ just FYI, our test fixtures do also do this "for free", e.g. [create_member](https://github.com/getsentry/sentry/blob/413e647dcae5c3e84534c14f31d361f792c0ce63/src/sentry/testutils/factories.py#L290)